releasenotes/notes/oracle_compliation-c5d75e0aaf04c8f1.yaml

```diff
@@ -0,0 +1,25 @@
+---
+features:
+  - |
+    This first version of an oracle compiler allows to "compile" typed python functions (operating only on bits (Int1) ) into quantum circuits.
+
+    .. jupyter-execute:
+
+      from qiskit.transpiler.oracle_compiler import compile_oracle
+      from qiskit.transpiler.oracle_compiler.types import Int1
```

Comment:
these two lines don't work. make sure the release notes is correct and runnable when all changes are done.